include/inexor/vulkan-renderer/vk_tools/enumerate.hpp

```diff
@@ -10,15 +10,22 @@ namespace inexor::vulkan_renderer::vk_tools {
 /// @note Because device layers are deprecated in Vulkan, we are not exposing the ``pLayerName`` parameter of
 /// ``vkEnumerateDeviceExtensionProperties`` as a parameter here
 /// @param physical_device The physical device to get all extension properties form
-/// @exception inexor::vulkan_renderer::VulkanException vkEnumerateDeviceExtensionProperties call failed
+/// @exception VulkanException vkEnumerateDeviceExtensionProperties call failed
 /// @return A std::vector of all device extension properties of a physical device (this can be empty!)
 [[nodiscard]] std::vector<VkExtensionProperties>
 get_all_physical_device_extension_properties(VkPhysicalDevice physical_device);
 
 /// Call vkEnumeratePhysicalDevices
 /// @param inst The Vulkan instance
-/// @exception inexor::vulkan_renderer::VulkanException vkEnumeratePhysicalDevices call failed
+/// @exception VulkanException vkEnumeratePhysicalDevices call failed
 /// @return A std::vector of all physical devices which are available on the system (this can be empty!)
 [[nodiscard]] std::vector<VkPhysicalDevice> get_all_physical_devices(VkInstance inst);
 
+/// Call vkGetPhysicalDeviceQueueFamilyProperties
+/// @param physical_device The physical device to get all extension properties form
+/// @exception VulkanException vkGetPhysicalDeviceQueueFamilyProperties call failed
+/// @return A std::vector of all queue families which are available on the system (this can be empty!)
+[[nodiscard]] std::vector<VkQueueFamilyProperties>
+get_all_physical_device_queue_family_properties(VkPhysicalDevice physical_device);
```

Comment:
This name is a little long :sweat_smile:, maybe could just be `get_queue_family_properties`?

Reply:
yes, maybe the `all` can be removed in all names?

Reply:
Done.